joomla/joomla-cms
# Move Content Modal History from mootools to bootstrap

## What this PR does

This PR convert the old Mootools modal to a Bootstrap modal for the Content History button

Before:
![before](https://cloud.githubusercontent.com/assets/2659866/9443310/a9c9e11c-4a80-11e5-908f-9c2485d07259.PNG)

After:
![after](https://cloud.githubusercontent.com/assets/2659866/9452030/5a6b1604-4ab2-11e5-8397-2375949b810f.PNG)
## How to test
1. Confirm all functions of content history are working like they have to do
2. Confirm the Bootstrap modal is also opening on the ContentHistory field type. You can test this to add the following code to a XML form:

```
<field type="contenthistory" name="contenthistory" label="Content History Test" />
```

Use the Global configuration for example, you can find this this form file here: `/administrator/components/com_config/model/form/application.xml`
